README.md

```diff
@@ -49,6 +49,7 @@ implement the api in the easiest way, depending on the current platform.
 | Native file chooser            | ✔       | ✔   | ✔       | ✔    | ✔     |
 | Notifications                  | ✔       |     | ✔       | ✔    | ✔     |
 | Orientation                    | ✔       |     |         |      |       |
+| Processors                     |         |     |         |      | ✔     |
```

Comment:
`processor` was present in this list prior to https://github.com/kivy/plyer/pull/483.

`processor` is now deprecated in favor of `cpu`, so I don't feel that should be in this list.